packages/scene-composer/src/interfaces/components.ts

```diff
@@ -36,10 +37,16 @@ export interface IModelRefComponent extends ISceneComponent {
 }
 
 export interface ISubModelRefComponent extends ISceneComponent {
+  [x: string]: any;
```

Comment:
Please do not use `any`. It removes type checking in TypeScript applications and it's effects can have widespread repercussions.